sdk/ml/azure-ai-ml/tests/dsl/unittests/test_command_builder.py

```diff
@@ -680,6 +680,27 @@ def test_resources_from_dict(self, test_command_params):
         rest_dict = command_node._to_rest_object()
         assert rest_dict["resources"] == {"instance_type": "STANDARD_D2"}
 
+    def test_queue_settings(self, test_command_params):
+        expected_queue_settings = {"job_tier": "standard"}
```

Comment:
`{"job_tier": "standard", "priority":"medium"}`
Can we please include `priorty` as part of the test?

Reply:
Done